src/erlcloud_aws.erl

```diff
@@ -363,6 +370,39 @@ config_env() ->
         _ -> {error, environment_config_unavailable}
     end.
 
+config_with_region(Config) ->
+    case {os:getenv("AWS_REGION"), application:get_env(erlcloud, aws_region)} of
```

Comment:
so we already have OS env, AWS profile, and config params. 
making it via application env is okay way but i'm not sure we need one more way.